internal/urlshortener/urlshortener.go

```diff
@@ -0,0 +1,106 @@
+package urlshortener
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+	"time"
+
+	valid "github.com/asaskevich/govalidator"
+)
+
+// User is a representation of a User. Dah.
+type shortURL struct {
+	ID            uint64
+	URL           string `json:"url,omitempty"`
+	VisitsCounter uint64
+}
+
+type shortURLService struct {
+	urlDatabase  shortURLStorage
+	makeFakeLoad bool
+}
+
+func (u *shortURLService) IsHealthy() (bool, error) {
+	return true, nil
+}
+
+func (u *shortURLService) generateFakeLoad(s string) error {
```

Comment:
It's weird to me that you write a 'u' because it's a service.

Reply:
:man_shrugging: i'll change it